<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellLineage</class>
 <widget class="QMainWindow" name="CellLineage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cell Lineage 9000</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons_LGPL/timespan.png</normaloff>:/Icons_LGPL/timespan.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QFrame" name="treeTextView">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3"/>
           </widget>
           <widget class="QFrame" name="frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="margin">
              <number>2</number>
             </property>
             <item>
              <widget class="QSplitter" name="splitter">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <widget class="QVTKWidget" name="vtkLineageViewWidget" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>1</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>150</height>
                 </size>
                </property>
               </widget>
               <widget class="QVTKWidget" name="vtkVolumeViewWidget" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>150</height>
                 </size>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Appearance Control</string>
              </property>
              <layout class="QVBoxLayout">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="margin">
                <number>9</number>
               </property>
               <item>
                <layout class="QVBoxLayout">
                 <property name="spacing">
                  <number>6</number>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <layout class="QHBoxLayout">
                   <property name="spacing">
                    <number>6</number>
                   </property>
                   <property name="margin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QCheckBox" name="radialLayoutCheckBox">
                     <property name="text">
                      <string>Radial</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="distanceCheckBox">
                     <property name="text">
                      <string>Scale Distance By Time</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="colorEdgesCheckBox">
                     <property name="text">
                      <string>Color By Time</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout">
                   <property name="spacing">
                    <number>6</number>
                   </property>
                   <property name="margin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QCheckBox" name="isoContourCheckBox">
                     <property name="text">
                      <string>IsoContour</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="collapseModeCheckBox">
                     <property name="text">
                      <string>Collapse Mode</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="labelsCheckBox">
                     <property name="text">
                      <string>Labels</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="backPlaneCheckBox">
                     <property name="text">
                      <string>Back Plane</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout">
                   <property name="spacing">
                    <number>6</number>
                   </property>
                   <property name="margin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>   Layout Angle</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="radialLayoutAngleSpinBox">
                     <property name="minimum">
                      <number>90</number>
                     </property>
                     <property name="maximum">
                      <number>360</number>
                     </property>
                     <property name="singleStep">
                      <number>10</number>
                     </property>
                     <property name="value">
                      <number>360</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label2">
                     <property name="text">
                      <string>Log Spacing</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="logSpacingSpinBox">
                     <property name="maximumSize">
                      <size>
                       <width>60</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="decimals">
                      <number>1</number>
                     </property>
                     <property name="minimum">
                      <double>0.100000000000000</double>
                     </property>
                     <property name="maximum">
                      <double>1.000000000000000</double>
                     </property>
                     <property name="singleStep">
                      <double>0.100000000000000</double>
                     </property>
                     <property name="value">
                      <double>1.000000000000000</double>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout">
                   <property name="spacing">
                    <number>6</number>
                   </property>
                   <property name="margin">
                    <number>8</number>
                   </property>
                   <item>
                    <widget class="QCheckBox" name="elbowCheckBox">
                     <property name="text">
                      <string>Elbow</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>Angle</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="elbowAngleSlider">
                     <property name="minimum">
                      <number>-100</number>
                     </property>
                     <property name="maximum">
                      <number>100</number>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="title">
               <string>Time Control</string>
              </property>
              <layout class="QVBoxLayout">
               <property name="spacing">
                <number>2</number>
               </property>
               <property name="margin">
                <number>2</number>
               </property>
               <item>
                <widget class="QSliderLineEdit" name="timeSlider" native="true"/>
               </item>
               <item>
                <widget class="QVCRWidget" name="vcr" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>300</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="geneTableView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenLineageFile"/>
    <addaction name="actionOpenDataFile"/>
    <addaction name="actionOpenGeneData"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenLineageFile"/>
   <addaction name="actionOpenDataFile"/>
   <addaction name="actionOpenGeneData"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOpenDataFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/FamFamFamIcons.qrc">
     <normaloff>:/icons/folder_image.png</normaloff>:/icons/folder_image.png</iconset>
   </property>
   <property name="text">
    <string>Open Volume Time Series...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Icons/FamFamFamIcons.qrc">
     <normaloff>:/icons/door_in.png</normaloff>:/icons/door_in.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpenLineageFile">
   <property name="icon">
    <iconset resource="Icons/FamFamFamIcons.qrc">
     <normaloff>:/icons/folder_page.png</normaloff>:/icons/folder_page.png</iconset>
   </property>
   <property name="text">
    <string>Open Lineage Tree...</string>
   </property>
   <property name="iconText">
    <string>Open Lineage Tree</string>
   </property>
   <property name="toolTip">
    <string>Open Lineage Tree</string>
   </property>
  </action>
  <action name="actionOpenGeneData">
   <property name="icon">
    <iconset resource="Icons/FamFamFamIcons.qrc">
     <normaloff>:/icons/folder_table.png</normaloff>:/icons/folder_table.png</iconset>
   </property>
   <property name="text">
    <string>Open Gene Expression Data...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSliderLineEdit</class>
   <extends>QWidget</extends>
   <header>QSliderLineEdit</header>
  </customwidget>
  <customwidget>
   <class>QVCRWidget</class>
   <extends>QWidget</extends>
   <header>QVCRWidget</header>
  </customwidget>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons/FamFamFamIcons.qrc"/>
 </resources>
 <connections/>
</ui>
